pelican-bootstrap3/templates/includes/comments.html

```diff
@@ -8,22 +8,6 @@
             /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
             var disqus_shortname = '{{ DISQUS_SITENAME }}'; // required: replace example with your forum shortname
 
-            {% if article %}
-                {% if not DISQUS_NO_ID %}
-                    var disqus_identifier = '{{ article.date|strftime('%Y-%m-') ~ article.slug if DISQUS_ID_PREFIX_SLUG else article.slug }}';
-                {% endif %}
-                var disqus_url = '{{ SITEURL }}/{{ article.url }}';
-            {% elif page %}
-                {% if not DISQUS_NO_ID %}
-                    var disqus_identifier = 'page-{{ page.slug }}';
-                {% endif %}
-                var disqus_url = '{{ SITEURL }}/{{ page.url }}';
-            {% endif %}
-
-            var disqus_config = function () {
-                this.language = "{{ DEFAULT_LANG }}";
-            };
-
```

Comment:
Please pardon my ignorance on this, but it seems like removing this block would break Disqus support.  We would have to update the function below to include the new embedded JS block like:

- https://help.disqus.com/customer/portal/articles/472098-javascript-configuration-variables